packages/connectors/src/coinbaseWallet.ts

```diff
@@ -17,37 +14,22 @@ import {
   numberToHex,
 } from 'viem'
 
-// TODO(@3): Set `enableMobileWalletLink` to `true`
+type Preference = Parameters<CoinbaseWalletSDK['makeWeb3Provider']>[0]
+
 export type CoinbaseWalletParameters = Evaluate<
   Mutable<
     Omit<
       ConstructorParameters<typeof CoinbaseWalletSDK>[0],
-      'reloadOnDisconnect' // remove property since TSDoc says default is `true`
-    >
-  > & {
-    /**
-     * Fallback Ethereum JSON RPC URL
-     * @default ""
-     */
-    jsonRpcUrl?: string | undefined
-    /**
-     * Fallback Ethereum Chain ID
-     * @default 1
-     */
-    chainId?: number | undefined
-    /**
-     * Whether or not to reload dapp automatically after disconnect.
-     * @default false
-     */
-    reloadOnDisconnect?: boolean | undefined
-  }
+      'appChainIds' // set via wagmi config
+    > & {
+      preference?: Preference
```

Comment:
Let's inline `Preference`

```suggestion
      preference?: Parameters<CoinbaseWalletSDK['makeWeb3Provider']>[0] | undefined
```